lnbits/tasks.py

```diff
@@ -19,6 +19,23 @@
 
 from .core import db
 
+tasks: List[asyncio.Task] = []
+
+
+def create_task(coro_or_fut):
+    task = asyncio.create_task(coro_or_fut)
+    tasks.append(task)
+    return task
+
+
+def create_permanent_task(func):
+    return create_task(catch_everything_and_restart(func))
+
+
+def cancel_all_tasks():
+    for task in tasks:
+        task.cancel()
```

Comment:
- loop body should be wrapped in `try-catch`

Reply:
`task.cancel` doesnt throw, even if its already canceled